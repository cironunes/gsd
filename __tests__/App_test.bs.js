'use strict';

var Jest = require("@glennsl/bs-jest/src/jest.js");
var React = require("react");
var ReactTestingLibrary = require("bs-react-testing-library/src/ReactTestingLibrary.bs.js");
var App$ReasonReactExamples = require("../src/App.bs.js");

Jest.test("Component renders", (function (param) {
        return Jest.Expect.toMatchSnapshot(Jest.Expect.expect((function (eta) {
                            return ReactTestingLibrary.render(undefined, undefined, eta);
                          })(React.createElement(App$ReasonReactExamples.make, { })).container));
      }));

/*  Not a pure module */
