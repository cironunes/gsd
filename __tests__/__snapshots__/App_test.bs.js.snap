// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component renders 1`] = `
<div>
  <form>
    <span>
      hi
    </span>
  </form>
  <form>
    <span>
      hi
    </span>
  </form>
</div>
`;
